app/views/events/_events.html.haml

```diff
@@ -4,4 +4,4 @@
       .row.grouped_events
         .medium-12.columns
           %h4= date
-        = render workshops
+        = render workshops
```

Comment:
Leave a blank line at the end of these files (as per the [POSIX standard](http://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline)). It's a relic of history but still good practice.

Unrelated we should add an [.editorconfig](http://editorconfig.org/) file to enforce that, but that's a different PR.